packages/babel-plugin-proposal-class-properties/src/index.js

```diff
@@ -201,7 +201,14 @@ export default declare((api, options) => {
 
     // Must be late evaluated in case it references another private field.
     return () =>
-      template.statement`MAP.set(REF, VALUE);`({
+      template.statement`
+        MAP.set(REF, {
+          // configurable is always false for private elements
+          // enumerable is always false for private elements
+          writable: true,
```

Comment:
We can save bytes be doing an explicit check for `=== false`. Then, we don't have to always include the `writable: true`.